spec/files/test-variables.xql

```diff
@@ -0,0 +1,6 @@
+xquery version "3.0";
+
+declare option exist:serialize "method=json media-type=text/javascript";
+
+(: return value of $variable that should have been set in the query parameters:)
```

Comment:
I would declare the variable as external here:
```xquery
declare variable $variable external;
```

Reply:
you're right, makes it clearer. also added the declaration in the readme example